Adds details to readme (#26)
nautobot/nautobot-plugin-chatops-ansible: README.md

## Summary

The Nautobot Chatops Ansible plugin extends the capabilities of the Nautobot Chatops plugin to include a new command available to the plugin. This is done by registering to the Python entry point in Nautobot Plugin Chatops, that provides functionality to the code written to interact with Ansible. This plugin introduces the following sub commands to the `ansible` command:

* get_dashboard
* get_inventory
* get_projects
* get_job_templates
* run_job_template
* get_jobs

### Dashboard

The dashboard command provides an Ansible Tower/AWX status dashboard. This gives a summary of:
* How many hosts are on the Tower system
* How many failed hosts (last job is failed)
* Total number of inventories
* How many inventories have failed
* How many project sync failures have occurred

### Inventory

The inventory sub-command provides for having inventory details. Providing what hosts are in what inventory.

### Get Projects

Get projects will gather information about the projects available within the Ansible Tower/AWX instance. Information such as the name, description, SCM URL, and SCM branch are provided.

> ### Job Templates

Reviewer: Get Job Templates

| **develop** | [![Build Status](https://www.travis-ci.com/nautobot/nautobot-plugin-chatops-ansible.svg?token=D7kytCzfCypoGoueSBqJ&branch=develop)](https://www.travis-ci.com/github/nautobot/nautobot-plugin-chatops-ansible) |

## Summary

Chatbot that allows users to communicate directly with Ansible AWX/Tower from various chat platforms.

The Nautobot Chatops Ansible plugin extends the capabilities of the Nautobot Chatops plugin to include a new command available to the plugin. This is done by registering to the Python entry point in Nautobot Plugin Chatops, that provides functionality to the code written to interact with Ansible. This plugin introduces the following sub commands to the `ansible` command:

* get_dashboard
* get_inventory
* get_projects
* get_job_templates
* run_job_template
* get_jobs

### Get Dashboard

The dashboard command provides an Ansible Tower/AWX status dashboard. This gives a summary of:
* How many hosts are on the Tower system
* How many failed hosts (last job is failed)
* Total number of inventories
* How many inventories have failed
* How many project sync failures have occurred

### Get Inventory

> The inventory sub-command provides for having inventory details. Providing what hosts are in what inventory.

Reviewer 2: This sentence did not read well to me.

The inventory sub-command provides a list of hosts in the selected inventory.

### Get Projects

Get projects will gather information about the projects available within the Ansible Tower/AWX instance. Information such as the name, description, SCM URL, and SCM branch are provided.

### Get Job Templates

> This gathers the information about the defined job templates on the Ansible Tower/AWX instance. It will provide the name, description, project, and the associated inventory with the Job Template configured.

Reviewer 2: It will provide the name, description, project, and the associated inventory for the selected Job Template.

### Run Job Template

> The natural progression of gathering job templates is then to execute a particular job template. If a templates does not need any extra vars or surveys to be completed, it will be able to be executed. The response from Ansible Tower/AWX of the job number will then be provided back via chat to know what job number was executed.

Reviewer 2: Perhaps reword the second sentence to:

Executes a Job Template that does not require `extra vars` or surveys to be completed.

> The natural progression of gathering job templates is then to execute a particular job template. If a templates does not need any extra vars or surveys to be completed, it will be able to be executed. The response from Ansible Tower/AWX of the job number will then be provided back via chat to know what job number was executed.

Reviewer 2: The Job ID for the executed job template will be provided via a chat response.

A plugin for [Nautobot](https://github.com/nautobot/nautobot) [Chatops Plugin](https://github.com/nautobot/nautobot-plugin-chatops/)

## Summary

The Nautobot Chatops Ansible plugin extends the capabilities of the Nautobot Chatops plugin to include a new command available to the plugin. This is done by registering to the Python entry point in Nautobot Plugin Chatops, that provides functionality to the code written to interact with Ansible. This plugin introduces the following sub commands to the `ansible` command:

* get_dashboard
* get_inventory
* get_projects
* get_job_templates
* run_job_template
* get_jobs

### Dashboard

The dashboard command provides an Ansible Tower/AWX status dashboard. This gives a summary of:

> * How many hosts are on the host

Reviewer 3: What does this mean?

Author: Good call!!! :)

* How many failed hosts (last job is failed)
* Total number of inventories
* How many inventories have failed
* How many project sync failures have occurred

### Inventory

> The inventory sub-command provides for having inventory details. Providing 

Reviewer 3: Can `Providing` be removed or does it need to be elaborated on?

Author: 👍🏻

> ### Projects

Reviewer 3: ```suggestion
### Get Projects
```
Since all of the other sub headings are the command name, I thought it'd make sense to format this the same way.

Get projects will gather information about the projects available within the Ansible Tower/AWX instance. Information such as the name, description, SCM URL, and SCM branch are provided.

### Job Templates

This gathers the information about the defined job templates on the Ansible Tower/AWX instance. It will provide the name, description, project, and the associated inventory with the Job Template configured.

### Run Job Template

> The natural progression of gathering job templates is then to execute a particular job template. For those that do not need any extra vars or surveys to be completed, will be able to be run. The response from Ansible Tower/AWX of the job number will then be provided back via chat to know what job number was executed.

Reviewer 3: ```suggestion
The natural progression of gathering job templates is then to execute a particular job template. If a templates does not need any extra vars or surveys to be completed, it will be able to be executed. The response from Ansible Tower/AWX of the job number will then be provided back via chat to know what job number was executed.
```
The 2nd sentence was a little confusing for me, so this is an idea to clarify it.

### Get Job

The get jobs sub command will ask Tower for the last number of jobs, with a default count of the last 10 jobs. This will give the status of the jobs including:

* Job ID
* Name of the Job
* User that launched the job
* Created
* Finish time
* Status of the job

## Installation

> ### Build Status

Reviewer 3: Can we just remove these sections?

Author: This looks weird. I am going to move Build Status to a new spot and keep installation. We should absolutely have installation docs. I like having them on the main page myself, but am open to having them move to an independent page that has navigation to it.

Reviewer 3: Sounds good to me
